Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenWindowControllerIOS.mm

```diff
@@ -1293,7 +1293,7 @@ - (void)exitFullScreen:(CompletionHandler<void()>&&)completionHandler
             _previewWindowController = nil;
         }
 
-        if (auto* manager = self._manager) {
+        if (!!self._manager) {
```

Comment:
Why the `!!`?
```suggestion
        if (self._manager)
```

Reply:
Because it's a pointer, not a bool.  The `!!` hints at that without looking at the definition of the getter.

But I'll change it.  (However, I can't take your suggested fix because you deleted the "`{`" which is needed for the body of the `if` statement.)